docs/provisioning/README.md

```diff
@@ -0,0 +1,52 @@
+# AzureML FL provisioning cookbook
+
+This page provides a "pick and choose" guide to provisioning a new FL environment. Each section provides multiple alternatives depending on your setup. Some sections are optional and can be skipped if you already have the required infrastructure in place.
+
+A lot of those steps are still marked :construction:, we are actively working to provide them im the coming iterations.
+
+## Prerequisites
+
+To enjoy these quickstart, you will need to:
+- have an active [Azure subscription](https://azure.microsoft.com) that you can use for development purposes,
+- have permissions to create resources, set permissions, and create identities in this subscription (or at least in one resource group),
+  - Note that to set permissions, you typically need _Owner_ role in the subscription or resource group - _Contributor_ role is not enough. This is key for being able to _secure_ the setup.
+- [install the Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+
+## Create an Azure ML Workspace
+
+Creating an Azure ML workspace is the starting point to create your full Federated Learning environment. Below are two options you could use, drawing from the existing documentation.
+
+| Tutorial | Description |
+| :-- | :-- |
+| [Docs](https://learn.microsoft.com/en-us/azure/machine-learning/quickstart-create-resources) | Create a workspace and then add compute resources to the workspace. You'll then have everything you need to get started with Azure Machine Learning. |
```

Comment:
Good move to link to existing public tutorials. Should we also point to a solution using the bicep file in our repo? (don't block the merge on that, it wouldn't add much and can always be done later)

Reply:
I'd rather use public docs because they provide lots of guidance outside of what our bicep does. Our script has just basic options. Let's consider creating a perfect bicep script for the azureml workspace and surface it there later. I'll add it as an feature request for Nov.